i18n/ar.toml

```diff
@@ -0,0 +1,195 @@
+# i18n strings for the Arabic (العربية) localization.
+
+[deprecation_warning]
+other = " documentation is no longer actively maintained. The version you are currently viewing is a static snapshot. For up-to-date documentation, see the "
+
+[deprecation_file_warning]
+other = "Deprecated"
+
+[objectives_heading]
+other = "Objectives"
+
+[cleanup_heading]
+other = "Cleaning up"
+
+[prerequisites_heading]
+other = "قبل ان تبدأ"
+
+[whatsnext_heading]
+other = "ماذا بعد"
+
+[feedback_heading]
+other = "Feedback"
+
+[feedback_question]
+other = "هل كانت هذه الصفحة مفيدة؟"
+
+[feedback_yes]
+other = "نعم"
+
+[feedback_no]
+other = "لا"
+
+[latest_version]
+other = "احدث اصدار."
+
+[version_check_mustbe]
+other = " يجب أن يكون خادم Kubernetes الخاص بك هو الإصدار"
+
+[version_check_mustbeorlater]
+other = "يجب أن يكون خادم Kubernetes في الإصدار أو بعده "
+
+[version_check_tocheck]
+other = "لفحص النسخة اضغط "
+
+[caution]
+other = "احذر:"
+
+[note]
+other = "ملاحظة:"
+
+[warning]
+other = "تحذير:"
+
+[main_read_about]
+other = "أقرأ عن"
+
+[main_read_more]
+other = "قراءة المزيد"
+
+[main_github_invite]
+other = "Interested in hacking on the core Kubernetes code base?"
```

Comment:
This is English. See https://github.com/kubernetes/website/pull/19758/files#r395832484